pkg/apis/autoscaling/v1/clusterautoscaler_types.go

```diff
@@ -29,12 +29,17 @@ type ClusterAutoscalerSpec struct {
 	PodPriorityThreshold *int32 `json:"podPriorityThreshold,omitempty"`
 
 	// BalanceSimilarNodeGroups enables/disables the
-	// `--balance-similar-node-groups` cluster-autocaler feature.
+	// `--balance-similar-node-groups` cluster-autoscaler feature.
 	// This feature will automatically identify node groups with
 	// the same instance type and the same set of labels and try
 	// to keep the respective sizes of those node groups balanced.
 	BalanceSimilarNodeGroups *bool `json:"balanceSimilarNodeGroups,omitempty"`
 
+	// BalancingIgnoredLabels sets the --balancing-ignore-label cluster-autoscaler option.
+	// This option specifies a list of labels that cluster autoscaler should ignore when
+	// considering node group similarity.
```

Comment:
Is this the name of the label or name/value pairs?

Reply:
it should just be the label name